lib/assets/chart_builder/main.js

```diff
@@ -286,7 +286,9 @@ export function init(ctx, payload) {
       if (elemClass) { element.classList.add(elemClass) };
       selectElem.append(element);
     }
-    if (elemClass !== "unavailable-option") {appendCountOption(selectElem);}
+    if (elemClass !== "unavailable-option" && selectElem.name !== "data_variable") {
```

Comment:
Just guessing

```suggestion
    if (channelFields.includes(selectElem.name)) {
```

Reply:
Though I think we still need the unavailable option check

Reply:
Yep! We need it for now. And again, Vue is coming to save us. ;)